resources/definitions/ultimaker_s6.def.json

```diff
@@ -0,0 +1,53 @@
+{
+    "version": 2,
+    "name": "UltiMaker S6",
+    "inherits": "ultimaker_s8",
+    "metadata":
+    {
+        "visible": true,
+        "author": "UltiMaker",
+        "manufacturer": "Ultimaker B.V.",
+        "file_formats": "application/x-ufp;text/x-gcode",
+        "platform": "ultimaker_s7_platform.obj",
+        "bom_numbers": [
+            10700
+        ],
+        "firmware_update_info":
+        {
+            "check_urls": [ "https://software.ultimaker.com/releases/firmware/5078167/stable/um-update.swu.version" ],
+            "id": 5078167,
+            "update_url": "https://ultimaker.com/firmware?utm_source=cura&utm_medium=software&utm_campaign=fw-update"
+        },
+        "first_start_actions": [ "DiscoverUM3Action" ],
+        "has_machine_quality": true,
+        "has_materials": true,
+        "has_variants": true,
+        "machine_extruder_trains":
+        {
+            "0": "ultimaker_s6_extruder_left",
+            "1": "ultimaker_s6_extruder_right"
+        },
+        "nozzle_offsetting_for_disallowed_areas": false,
+        "platform_offset": [
+            0,
+            0,
+            0
+        ],
+        "platform_texture": "UltimakerS6backplate.png",
+        "preferred_material": "ultimaker_pla_blue",
+        "preferred_variant_name": "AA+ 0.4",
+        "quality_definition": "ultimaker_s8",
+        "supported_actions": [ "DiscoverUM3Action" ],
+        "supports_material_export": true,
+        "supports_network_connection": true,
+        "supports_usb_connection": false,
+        "variants_name": "Print Core",
+        "variants_name_has_translation": true,
+        "weight": -2
+    },
+    "overrides":
+    {
+        "adhesion_type": { "value": "'brim'" },
+        "machine_name": { "default_value": "Ultimaker S6" }
```

Comment:
```suggestion
        "machine_name": { "default_value": "UltiMaker S6" }
```

Reply:
Note: this is consistent with S8, are they both wrong ?

Reply:
No should be UltiMaker. I will adjust.